fsd_config/form_jsons/cof_r3/en/local-support.json

```diff
@@ -0,0 +1,92 @@
+{
+  "metadata": {},
+  "startPage": "/local-support",
+  "backLinkText": "Go back to application overview",
+  "pages": [
+    {
+      "title": "Local support",
+      "path": "/local-support",
+      "components": [
+        {
+          "name": "gqavTE",
+          "options": {},
+          "type": "Para",
+          "content": "<label class=\"govuk-!-font-weight-bold\">In this section, we'll ask about:\n</label>\n\n<ul class=\"govuk-list govuk-list--bullet\">\n<li>local support for your project</li>\n</ul>"
+        }
+      ],
+      "next": [
+        {
+          "path": "/your-support-for-the-project"
+        }
+      ]
+    },
+    {
+      "path": "/your-support-for-the-project",
+      "title": "Your support for the project",
+      "components": [
+        {
+          "name": "tDVPnl",
+          "options": {
+            "maxWords": "500"
+          },
+          "type": "FreeTextField",
+          "title": "Tell us about the local support for your project",
+          "hint": "This can include:\n<p>\n<ul class=\"govuk-hint\">\n<li>surveys of the community</li>\n<li>minutes of town hall meetings you have run</li>\n<li>the estimated number of volunteers you have for your project</li>\n<li>a letter from your local MP</li>\n</ul>\n</p>\n\nIf you plan to upload any supporting evidence below, reference it here."
+        },
+        {
+          "name": "bDWjTN",
+          "options": {
+            "required": false,
+            "optionalText": false
+          },
+          "type": "FileUploadField",
```

Comment:
I'll suggest changes or raise a PR to change these to the new upload component after retro